packages/vite/src/node/index.ts

```diff
@@ -66,7 +66,12 @@ export type {
   DevEnvironmentOptions,
   ResolvedDevEnvironmentOptions,
 } from './config'
-export type { Plugin, PluginOption, HookHandler } from './plugin'
+export type {
+  Plugin,
+  PluginOption,
+  HookHandler,
+  CustomPluginOptionsVite,
```

Comment:
Would we be able to not export this for now? I'm thinking it would be better to support it like https://github.com/vitejs/vite/blob/00deea4ff88e30e299cb40a801b5dc0205ac913d/packages/vite/types/metadata.d.ts once Rollup fixes it upstream.

Reply:
Sounds good 👍